### DISCUSSION TITLE
Unable to specify download time

### DISCUSSION THREAD

**Discussion by Tydoki** [MacielG1/VodRecovery#9] [Q&A]
When I try to specify a download time now the program just stalls after I hit enter and doesn't write any further text. 
![WindowsTerminal_wFouqUJvpd](https://github.com/MacielG1/VodRecovery/assets/49407654/32549a0d-cd89-43c5-8a60-8cae4d7c6ee2)

**Comment by MacielG1** [comment]
I tested it myself, and it's working. I think the issue is that it's downloading in the background, so you can't see the progress, because the part you're trying to trim is long. Try downloading just 10 seconds of it to test it. If it does, just wait for the whole thing to download.
In the next update I'll add some sort of output while downloading these long streams

**Reply by Tydoki** [reply]
Ah yes it is working just can't see output till it is done. 

Thanks
